test/typecheck/main.go

```diff
@@ -41,12 +41,15 @@ import (
 )
 
 var (
-	verbose   = flag.Bool("verbose", false, "print more information")
-	cross     = flag.Bool("cross", true, "build for all platforms")
-	platforms = flag.String("platform", "", "comma-separated list of platforms to typecheck")
-	timings   = flag.Bool("time", false, "output times taken for each phase")
-	defuses   = flag.Bool("defuse", false, "output defs/uses")
-	serial    = flag.Bool("serial", false, "don't type check platforms in parallel")
+	verbose    = flag.Bool("verbose", false, "print more information")
+	cross      = flag.Bool("cross", true, "build for all platforms")
+	platforms  = flag.String("platform", "", "comma-separated list of platforms to typecheck")
+	timings    = flag.Bool("time", false, "output times taken for each phase")
+	defuses    = flag.Bool("defuse", false, "output defs/uses")
+	serial     = flag.Bool("serial", false, "don't type check platforms in parallel")
+	skipTest   = flag.Bool("skip-test", false, "don't type check test code")
+	tags       = flag.String("tags", "", "comma-separated list of additional build tags to apply")
```

Comment:
clarify what this is in addition to (presumably os/arch-level build tags?)

Reply:
ACK -- it's in addition to go's defaults, we're not setting any ourselves other than these.

Reply:
I'm adding that this is in addition to the defaults, but I don't think we should document the defaults here as they come from go and could change (currently GOOS and GOARCH)

Reply:
(I think also more that are not as explicitly documented though?)